doc/user: spell out how to file support ticket (#28063)
MaterializeInc/materialize: doc/user/content/support.md

## Getting help

### Trials

If you run into a snag or need support during the trial period, join the [Materialize Slack community](https://materialize.com/s/chat) or [open an issue](https://github.com/MaterializeInc/materialize/issues/new/choose). Slack is the best place to get timely help from Materialize employees and other users!

### Customers

If you're a paying customer, you can access Materialize support via a shared
channel within your company's Slack or Microsoft Teams workspace.

To file a support ticket:

> 1. Send a message to your shared channel describing your issue.

Reviewer: ```suggestion
1. Send a message to the shared channel describing your issue.
```

2. Apply the `:ticket:` (🎫) emoji to your message.
3. Click the **File a ticket** button that appears.

> 4. Select the appropriate priority for your ticket. Consult your support
>    contract to determine the definition of each priority level and the
>    associated guaranteed response time.

Reviewer: ```suggestion
4. Select the appropriate priority for your ticket. You can find the definition
    of each priority level and the associated guaranteed response time in your
    support contract.
```

Author: Refactored away to refer to a table in this document.

> 5. Materialize will respond to the ticket within the response time outlined
>    in your support contract.

Reviewer: ```suggestion
5. The Materialize team will respond to the ticket within the response time
   associated with its priority.
```
